src/backend/backend.h

```diff
@@ -60,23 +60,55 @@ namespace snmalloc
 
     SNMALLOC_REQUIRE_CONSTINIT
     static inline FlatPagemap<MIN_CHUNK_BITS, PageMapEntry, PAL, fixed_range>
-      pagemap;
+      _pagemap;
```

Comment:
I really don't like having a thing called `pagemap` and a thing called `_pagemap`.  The only other place, I think, where we have underscore-prefixed things are when they're SFINAE template parameters.  Can we make this a bit more descriptive, for example `pagemapData` or `concretePagemap` or similar?

Should this be a field on `Pagemap`?  It needs to be initialised externally, but that could be either via a method call or by making it public.

Reply:
Or by keeping it `private` but adding `BackendAllocator` as a `friend`?  That way if the underlying data structure changes there's still just the existing call sites to change and no new intermediation.